src/applications/coronavirus-research/widgets/HeightWidget.jsx

```diff
@@ -13,18 +13,24 @@ function getEmptyState() {
     },
   };
 }
-
+function setLabelStyles(newState) {
+  let labelMarginLeftClass = 'vads-u-margin-left--1';
+  if (
+    newState &&
+    newState.touched.feet &&
+    newState.touched.inches &&
+    (!newState.value.feet || !newState.value.inches)
```

Comment:
can this be simplified with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining ?